### DISCUSSION TITLE
Another dictionary question: `{l+}` and `{l-}`.

### DISCUSSION THREAD

**Discussion by mattfbacon** [opensteno/plover#1606] [Q&A and Support] [answered]
I found these in the entry `"P-RBGS/H-RBGS/TK-RBGS": "P{l+}h{l-}.D",`. What do they do?

**Comment by stenoknight** [comment, marked as answer]
That's Eclipse syntax also. I believe it makes it so that the individual
letters of the entry won't be broken apart by the automatic linebreaking
feature of the transcript software.
